HotFix.Specification/logout/a_logout.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using HotFix.Core;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace HotFix.Specification.logout
+{
+    [TestClass]
+    public class a_logout
```

Comment:
It would be better to split the tests into two scenarios:

- an_initiated_logout
    - sends_a_logout_message_and_disconnects
- an_uninitiated_logout
    - sends_a_logout_message_and_disconnects